ui/lib/kubernetes/addon/templates/roles/role/credentials.hbs

```diff
@@ -1 +1 @@
-Role Credentials
\ No newline at end of file
+<Page::Credentials @role={{this.model.roleModel}} @backend={{this.model.backend}} />
```

Comment:
I mentioned this in the route but if you go forward with the suggestion then `@role` would be changed to `this.model` and you could remove the `@backend` arg and access that value on the role model in the component.